### DISCUSSION TITLE
震惊！［巡猎］岚和［丰饶］药师居然………

### DISCUSSION THREAD

**Discussion by love262-wkx** [share121/inter-knot#4816] [崩坏：星穹铁道]
大家信我，这是真的，真是太有乐子了
![img1145204867_1726149385327](https://github.com/user-attachments/assets/47776cc8-ad9c-44e7-88fb-99a550775e38)

**Comment by lllliufeng** [comment]
啊？<img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" >

**Comment by Robin-txdy** [comment]
好抽象<img src="https://img-static.mihoyo.com/communityweb/upload/03c78415a3318d527d307c52a188a5ad.png" alt="米游姬-呆滞" width="75" >

**Comment by arlcius** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >这下看懂了

**Comment by 3283276594** [comment]
啊？

**Comment by RocketsRat** [comment]
这还是空洞吗？给我干哪来了😨

**Comment by beiya0221** [comment]
这很难评，希望乐子神不会被妖弓祸祖一箭射爆
